addons/survey/static/src/js/survey_result.js

```diff
@@ -471,6 +475,28 @@ publicWidget.registry.SurveyResultWidget = publicWidget.Widget.extend({
         window.location.href = window.location.pathname + '?' + params.toString();
     },
 
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onQuestionFilterClick: function (ev) {
+        var cell = $(ev.target);
+        var row_id = cell.data('row_id') | 0;
```

Comment:
Guidelines: variables in JS need to be camelCased.

Also, you meant '||' to set a default value right? Not sure we want to play with bit operations here ^^

Reply:
OK, I'll update the existing function too.